core/types.mli

```diff
@@ -37,12 +37,28 @@ end
 
 module Policy : sig
   type kind_policy = Default | Full | Hide
+
+  module EffectSugar : sig
+    type opt = PresenceOmit | AliasOmit | ArrowsShowTheOneEffect | ArrowsCurriedCollectionAssumeFresh | ContractOperationArrows | OpenDefault | DifferentOperationArrows
+    type t = opt list
+    val default : unit -> t
+
+    val presence_omit              : t -> bool
+    val alias_omit                 : t -> bool
+    val arrows_show_the_one        : t -> bool
+    val arrows_collection_fresh    : t -> bool
+    val contract_operation_arrows  : t -> bool
+    val open_default               : t -> bool
+    val different_operation_arrows : t -> bool
```

Comment:
We don't want this option now as there is no surface syntax to match the printing. I want to maintain the invariant that anything we print is parseable.

Reply:
Created the syntax here.